app/scripts/angular-apimock.js

```diff
@@ -234,7 +260,8 @@ angular.module('apiMock', [])
 
 			// replace apiPath with mockDataPath.
 			var oldPath = req.url;
-			var redirectedPath = req.url.replace(config.apiPath, config.mockDataPath);
+
+			var redirectedPath = req.url.replace(apiPathMatched(req.url, config.apiPath), config.mockDataPath);
```

Comment:
It would have to become a object map of some kind. Otherwise mismatching array lengths would be really annoying... My idea was always to keep this library as simple as possible. And since it's mapping to physical files then having to move or change folder paths because the API changes is annoying. So let's go with this as you've done it and see what feedback we get. I'm planning on a major restructuring of configs and flags for `1.0.0` anyway.